.github/workflows/run-e2e-regression.yml

```diff
@@ -0,0 +1,59 @@
+name: Run E2E Regression Tests
+on:
+  schedule:
+    - cron: "0 0 * * 1-5"
+jobs:
+  run_e2e_tests:
+    name: Run E2E Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        package: ["ct", "bp"]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+
+      - name: Get cached dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cache/Cypress
+            **/node_modules
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Build tools
+        run: yarn build:${{ matrix.package }}
+
+      - name: Run cypress tests
+        uses: cypress-io/github-action@v2.8.2
+        with:
+          start: yarn dev:${{ matrix.package }}
+          wait-on: 'http://localhost:3000'
+          command: yarn run test:e2e:${{ matrix.package }}:hl
+          browser: chrome
+          headless: true
+          install: false
+        env:
+          CYPRESS_INCLUDE_TAGS: regression
+
+      - name: Generate report
+        if: ${{ always() }}
```

Comment:
Is `if` required it we use `always()`?

Reply:
AFAIK if is required in case the previous step fails - this way even if tests fail, we still generate and upload the report.